packages/lib/delivery-api/src/auth-http.ts

```diff
@@ -0,0 +1,61 @@
+import axios from 'axios';
+import {
+    Account,
+    getDeliveryServiceClient,
+    normalizeEnsName,
+} from 'dm3-lib-profile';
+
+import { ethers } from 'ethers';
+import { checkAccount } from './utils';
+
+const AUTH_SERVICE_PATH = process.env.REACT_APP_BACKEND + '/auth';
```

Comment:
Is this supposed to be used within both FE and BE ? If it might make sense to rename the REACT_APP_BACKEND var